Dockerfile

```diff
@@ -991,7 +991,9 @@ if [[ -n "${_PIP_ADDITIONAL_REQUIREMENTS=}" ]] ; then
     >&2 echo "         the container starts, so it is onlny useful for testing and trying out"
     >&2 echo "         of adding dependencies."
     >&2 echo
-    pip install --root-user-action ignore --no-cache-dir ${_PIP_ADDITIONAL_REQUIREMENTS}
+    pip install --root-user-action ignore --no-cache-dir \
+        ${ADDITIONAL_PIP_INSTALL_FLAGS} \
+        ${_PIP_ADDITIONAL_REQUIREMENTS}
```

Comment:
If we’re separating the two I wonder if it’s a good idea to disallow options in `_PIP_ADDITIONAL_REQUIREMENTS` by adding a `--` before it.